test/e2e/common.go

```diff
@@ -221,3 +225,61 @@ func buildKustomizeManifest(source string) ([]byte, error) {
 	}
 	return resources.AsYaml()
 }
+
+func executeSSHCommand(client *ssh.Client, command string) (string, error) {
+	session, err := client.NewSession()
+	if err != nil {
+		return "", fmt.Errorf("failed to create SSH session: %v", err)
+	}
+	defer session.Close()
+
+	output, err := session.CombinedOutput(command)
+	if err != nil {
+		return "", fmt.Errorf("failed to execute command '%s': %v", command, err)
+	}
+
+	return string(output), nil
+}
+
+// ParseFileSystemOutput parses the given filesystem output string and checks if booted from disk
+func ParseFileSystemOutput(output string) bool {
+	lines := strings.Split(output, "\n")
+
+	for _, line := range lines[1:] { // Skip header line
+		if line == "" {
+			continue
+		}
+
+		fields := strings.Fields(line)
+		if len(fields) < 6 {
+			continue // Skip malformed lines
+		}
+
+		if fields[5] == "/" && !strings.Contains(fields[0], "tmpfs") {
+			return true // Found a non-tmpfs root filesystem
+		}
+	}
+
+	return false
+}
+
+func IsBootedFromDisk(client *ssh.Client) (bool, error) {
```

Comment:
Please add a doc string

Reply:
Is what I now implemented similar to what you had in mind?